DSharpPlus/Entities/Guild/DiscordGuild.cs

```diff
@@ -461,6 +474,95 @@ internal DiscordGuild()
+        /// <param name="description">The optional description of the sticker.</param>
+        /// <param name="emoji">The emoji to associate the sticker with.</param>
+        /// <param name="format">The file format the sticker is written in.</param>
+        /// <param name="file">The sticker.</param>
+        /// <returns>The created sticker.</returns>
+        public async Task<DiscordMessageSticker> CreateStickerAsync(string name, string? description, DiscordEmoji emoji, StickerFormat format, Stream file)
+        {
+            if (emoji.Id is not 0)
+                throw new InvalidOperationException("Only unicode emoji can be used for stickers.");
+
+            if (name.Length is > 30 or < 2)
+                throw new ArgumentOutOfRangeException(nameof(name), "Sticker name must be 2-30 characters.");
+
+            if (description?.Length > 100)
+                throw new ArgumentOutOfRangeException(nameof(description), "Sticker description must be less than or equal to 100 characters.");
+
+            var filename = format switch
+            {
+                StickerFormat.PNG => "sticker.png",
+                StickerFormat.APNG => "sticker.apng",
+                StickerFormat.LOTTIE => "sticker.json"
+            };
+
+            return await this.Discord.ApiClient.CreateGuildStickerAsync(this.Id, name, description, emoji.Name, new($"sticker.{format}", file, null)).ConfigureAwait(false);
+        }
+
+        /// <summary>
+        /// Modifies a sticker. Requires ManageStickers permission.
+        /// </summary>
+        /// <param name="stickerId">The Id of the sticker to modify.</param>
+        /// <param name="name">The name of the sticker</param>
+        /// <param name="description">The description of the sticker</param>
+        /// <param name="emoji">The emoji to associate with this sticker.</param>
+        /// <returns>The modified sticker.</returns>
+        /// <exception cref="InvalidOperationException"></exception>
+        public async Task<DiscordMessageSticker> ModifyStickerAsync(ulong stickerId, string? name, string? description, DiscordEmoji? emoji)
+        {
+            if (emoji?.Id is not null and > 0)
+                throw new InvalidOperationException("Only unicode emojis can be used with stickers.");
+
+            var sticker = await this.Discord.ApiClient.ModifyStickerAsync(this.Id, stickerId, name, description, emoji.Name).ConfigureAwait(false);
+
+            if (this._stickers.TryGetValue(stickerId, out var old))
+                this._stickers.TryUpdate(stickerId, sticker, old);
+
+            return sticker;
+        }
+
+        /// <summary>
+        /// Gets the current guild's stickers.
+        /// </summary>
+        /// <returns>The stickers on teh current guild, if any.</returns>
+        public async Task<IReadOnlyList<DiscordMessageSticker>> GetStickersAsync()
+        {
+            var stickers = await this.Discord.ApiClient.GetStickersAsync(this.Id);
```

Comment:
```suggestion
            var stickers = await this.Discord.ApiClient.GetGuildStickersAsync(this.Id);
```

Reply:
That nomenclature is redundant; the method is already on a guild